message-channel/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: pattern
+title: Message Channel
+folder: message-channel
+permalink: /patterns/message-channel/
+categories: Integration
+tags: Java
+---
+
+**Intent:** When two applications communicate using a messaging system they do it by using logical addresses
+of the system, so called Message Channels.
+
+![alt text](./etc/message-channel.png "Message Channel")
```

Comment:
Doesn't ``![Message Channel](./etc/message-channel.png)`` produces the same result? It is clearer, shorter and more readable.